### PR TITLE
Clarify definition of "network health."

### PR DESCRIPTION
**Pull request by mtrippled**
## High Level Overview of Change
The existing documentation describes network health at a very high level with no nuance that reflects the reality. This update better defines network health as well as provides context about related factors.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [X] Documentation Updates
- [ ] Release

### CODE REVIEW

**Review by ximinez — APPROVED** [review]

**Review by Bronek — CHANGES_REQUESTED** [review]

**Review by HowardHinnant — APPROVED** [review]

File: src/ripple/app/misc/FeeEscalation.md
Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md:206]: Duration is more precise, though "time" as used here means duration among other things. So duration is more appropriate here and I changed other instances of "time" to reflect that.

**Comment by mtrippled** [comment]
@ximinez I'd like to cover this PR with you next week, please. There are some things about fees that could probably be clarified a bit about network health. Thanks for reviewing this. Let's not merge this until then.

Also, @Bronek I'm doing something here that's generally not OK--making changes to a PR after it's submitted. But this is a small PR, just for documentation. Anyway, normally what we do is submit a PR once we think it's feature complete, and only make changes based on review suggestions. So, "do as I say not as I do," please. :-)

**Review by Bronek — APPROVED** [review]

**Comment by intelliot** [comment]
> I'm doing something here that's generally not OK--making changes to a PR after it's submitted. But this is a small PR, just for documentation. Anyway, normally what we do is submit a PR once we think it's feature complete, and only make changes based on review suggestions.

I think most PRs are actually a little more flexible on this point - it's fine to make (justified) changes to a PR after it's submitted, but it _does_ mean that the PR will generally need re-review/re-approval before merging. That is perfectly OK though.

**Comment by intelliot** [comment]
Given

> Let's not merge this until then.

I've set this PR to "draft" status to ensure it isn't merged until deemed ready.

**Comment by mtrippled** [comment]
@Bronek @HowardHinnant @ximinez @intelliot I just refined the document a bit more, and fixed a typo. Please scan again.

File: src/ripple/app/misc/FeeEscalation.md
Comment (Bronek) [inline src/ripple/app/misc/FeeEscalation.md:218]: Should this sentence be moved up ? It seems disconnected here.

Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md]: I just removed the sentence entirely. It's not a useful detail here.

**Review by Bronek — APPROVED** [review]

**Comment by intelliot** [comment]
note: this PR has changed since ximinez's last review, so it needs a re-review.

File: src/ripple/app/misc/FeeEscalation.md
Comment (ximinez) [inline src/ripple/app/misc/FeeEscalation.md:214]: This last sentence feels out of place. Should it be at the top in a paragraph of its own (as an intro)? Or as a conclusion? Or just removed entirely?

**Review by ximinez — CHANGES_REQUESTED** [review]
I'm ok with this as it is, but I think it could be better with a little clarification of the last sentence.

File: src/ripple/app/misc/FeeEscalation.md
Comment (ckeshava) [inline src/ripple/app/misc/FeeEscalation.md:213]: So the consensus process takes >20 seconds, although no transactions were included in the ledger.

Can we list any factors that might cause this issue? Historically, have such problems occurred on the mainnet or other affiliated blockchain networks? Can we provide a link to such an example?

File: src/ripple/app/misc/FeeEscalation.md
Comment (ckeshava) [inline src/ripple/app/misc/FeeEscalation.md:197]: Can we use the [Reliability Score](https://www.viator.com/account/booking/144285441/1382799619/994751177) as a proxy for measuring the network health? It seems to indicate the degree of similarity in the calculations between validators.

File: src/ripple/app/misc/FeeEscalation.md
Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md]: fixed

File: src/ripple/app/misc/FeeEscalation.md
Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md:213]: That goes beyond clarifying what stability is and gets into speculation and diagnostics.

File: src/ripple/app/misc/FeeEscalation.md
Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md:197]: I don't know what that is, and it brings me to a site that asks for my email address.

File: src/ripple/app/misc/FeeEscalation.md
Comment (ckeshava) [reply src/ripple/app/misc/FeeEscalation.md:213]: I feel giving such examples would provide more clarity. As it stands, the reader does not understand why the network could become unstable

File: src/ripple/app/misc/FeeEscalation.md
Comment (ckeshava) [reply src/ripple/app/misc/FeeEscalation.md:197]: Sorry, the link is wrong. Here's the reference: https://xrpl.org/negative-unl.html#reliability-measurement
image

Comment (mtrippled) [reply src/ripple/app/misc/FeeEscalation.md:197]: @ckeshava 
The problem with the existing document is that it can be mis-interpreted to mean that 5s latency in consensus is some extreme upper limit, beyond which the network is in a faulty state. This PR corrects the language and hopefully encourages approaching the issue with some nuance. I didn't really intend this to be an exhaustive treatment of all the ways that the network can have problems, or different diagnostics and measurements that can be done. That is actually quite a sizable topic. But for now I prefer that this stays concise and mainly clarifies the original statement.

**Review by ximinez — APPROVED** [review]

**Comment by intelliot** [comment]
@ckeshava for the ideas + open questions that you have, please feel free to open a new issue (or better - a PR with your proposed changes). They are likely outside the scope of this particular PR

**Pull request merged**
